Makefile

```diff
@@ -73,7 +74,7 @@ init:
 
 vc-scheduler: init
 	if [ ${SUPPORT_PLUGINS} = "yes" ];then\
-		CC=${CC} CGO_ENABLED=1 go build -ldflags ${LD_FLAGS} -o ${BIN_DIR}/vc-scheduler ./cmd/scheduler;\
+		CC=${MUSL_CC} CGO_ENABLED=1 go build -ldflags ${LD_FLAGS_CGO} -o ${BIN_DIR}/vc-scheduler ./cmd/scheduler;\
```

Comment:
musl-gcc is used by default? I'm not sure if this is a good idea.

If this modification is necessary, we should first check whether musl-gcc is installed and give a prompt

Reply:
Yeah when we use CGO and custom plugin, musl-gcc is required, so I have add the recommended way that built with docker in `Build the image and plugin in `Docker`(Recommended):`.